de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/grid/GridViewPresenterImpl.java

```diff
@@ -568,9 +569,10 @@ public void setParentPresenter(DiskResourceView.Presenter parentPresenter) {
 
     @Override
     public void setSelectedDiskResourcesById(List<? extends HasId> diskResourcesToSelect) {
-        SelectDiskResourceByIdStoreAddHandler diskResourceByIdStoreAddHandler = new SelectDiskResourceByIdStoreAddHandler(diskResourcesToSelect,
-                                                                                                                          this);
-        HandlerRegistration diskResHandlerReg = listStore.addStoreAddHandler(diskResourceByIdStoreAddHandler);
+        SelectDiskResourceByIdStoreAddHandler diskResourceByIdStoreAddHandler =
```

Comment:
If I select that line and do `Code -> Reformat code`. It does some formatting. But sometimes the formatting looks odd. But when I select the whole file and do the same thing, then formatting looks different.